graph/db/graph_test.go

```diff
@@ -94,6 +94,37 @@ func createTestVertex(t testing.TB) *models.LightningNode {
 	return createLightningNode(priv)
 }
 
+// TestNodeCRUD tests the basic CRUD operations for a LightningNode. This
+// includes checking that a node entry gets created if a Channel Edge is added
+// that references a node that does not yet exist.
+func TestNodeCRUD(t *testing.T) {
```

Comment:
It seems that we have this test case already, named as `TestAddChannelEdgeShellNodes`. It also covers fetching the shell node (this is also how i found it).